lava-slave/Dockerfile

```diff
@@ -6,6 +6,7 @@ RUN apt-get update
 COPY scripts/stop.sh .
 COPY scripts/start.sh .
 
+# e2fsprogs is for libguestfs
```

Comment:
Looks like a stray comment.

Reply:
No, I add a comment missing like the one on lava-master/Dockerfile.
But e2fsprogs is already present on lava-slave/Dockerfile.
So it's not a stray comment.

Reply:
OK, thanks for clarifying.